src/main/kotlin/lotto/view/InputView.kt

```diff
@@ -1,24 +1,53 @@
 package lotto.view
 
 import lotto.domain.Lotto
+import lotto.domain.LottoPurchaseRequest
 
 class InputView {
 
-    fun inputPurchaseAmount(): Int {
+    fun createRequest(): LottoPurchaseRequest {
+        val inputMoney = inputPurchaseAmount()
+        val purchaseAmount = getPurchasableNum(inputMoney)
+        var manualAmount = inputManualAmount()
+
+        while (manualAmount > purchaseAmount) {
+            println("구입금액은 0보다 크고 100만보다 작거나 같아야 합니다. 다시 입력해주세요.")
+            manualAmount = inputManualAmount()
+        }
+
+        val manualNumbers = inputManualNumbers(manualAmount)
+        val autoAmount = purchaseAmount - manualAmount
+
+        return LottoPurchaseRequest(autoAmount, manualNumbers)
+    }
+
+    private fun inputPurchaseAmount(): Int {
         println("구입금액을 입력해 주세요.")
         var inputAmount = readNumber()
-        while (!isValid(inputAmount)) {
+        while (!(inputAmount in PURCHASABLE_RANGE)) {
             println("구입금액은 0보다 크고 100만보다 작거나 같아야 합니다. 다시 입력해주세요.")
             inputAmount = readNumber()
         }
         return inputAmount
     }
 
+    private fun inputManualAmount(): Int {
+        println("수동으로 구매할 로또 수를 입력해 주세요.")
+        return readNumber()
+    }
+
+    private fun inputManualNumbers(cnt: Int): List<List<Int>> {
+        println("수동으로 구매할 번호를 입력해 주세요.")
+        val result = mutableListOf<List<Int>>()
+        repeat(cnt) {
+            result += inputLottoNumbers()
+        }
+        return result
```

Comment:
다음 코드처럼 List를 빌드할 수도 있습니다🙂
```suggestion
        return List(cnt) {
            inputLottoNumbers()
        }
```

Reply:
아 이런 방법이 있는걸 잊고 있었네요